Modify requirements content

rjrpaz opened the pull request.

pierre commented:
@rjrpaz Could you take a look at the `docker_ci` job errors? Wondering if the Docker build needs to be updated too?

`kpm_ci` job failure is tracked by https://github.com/killbill/killbill-cloud/issues/254

rjrpaz commented:
- docker_ci error is related to the same thing about trying to update the ansible collection using https. I fixed but the pipeline tries to download the repo using a branch that does not exist (my user was not able to do new branches in the original repo so I had to a fork and create the new repo there). If you run this pipeline from the original repo and the branch exists, that steps should work.
- kpm_ci failed because of a ruby gems update trying to install a higher version. I also updated that but now the pipeline fails as part of unit testing. Your team members will probably know better about these errors.
